.goreleaser.yaml

```diff
@@ -77,6 +77,18 @@ builds:
       - windows
       - darwin
 
+  - id: plugin_nist_nvd
+    main: ./internal/nistnvd/cmd
+    binary: "nist_nvd@{{ .Version }}"
```

Comment:
I don't have a strong preference, but it might be nicer to match code path to the plugin name and have `internal/nist_nvd`

Reply:
This is go package name based on "Effective Go" practices:
https://go.dev/doc/effective_go#package-names
> ... By convention, packages are given lower case, single-word names; there should be no need for underscores or mixedCaps.

Plugin name is still kept as `blackstork/nist_nvd`

Reply:
understood!